src/polyfem/assembler/MassMatrixAssembler.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <polyfem/assembler/ElementAssemblyValues.hpp>
+#include <polyfem/assembler/AssemblyValsCache.hpp>
+#include <polyfem/assembler/MatParams.hpp>
+
+#include <Eigen/Sparse>
+#include <vector>
+#include <iostream>
+#include <cmath>
+#include <memory>
+
+// mass matrix assembler
+namespace polyfem::assembler
+{
+	class MassMatrixAssembler
+	{
+	public:
+		/// @brief Assembles the mass matrix.
+		/// @param[in]  is_volume True if the mesh is volumetric.
```

Comment:
do we need these?

Reply:
Oh, right I restored this file because it was where I implemented the L2-projection matrix. I can delete the unused assemble function for the mass matrix.

Reply:
I am going to leave it because it would be a lot to refactor to use the new version.